src/v/kafka/server/member.h

```diff
@@ -142,6 +144,9 @@ class group_member {
     /// Check if the member is syncing.
     bool is_syncing() const { return bool(_sync_promise); }
 
+    /// Check if member is static
+    bool is_static() const { return bool(_state.instance_id); }
```

Comment:
EDIT: i saw once I got to the tests that indeed it can be mixed

```
    @cluster(num_nodes=6)
    def test_mixed_consumers_join(self):
        """
        Test validating that dynamic and static consumers may exists in the same group
        """
```

Reply:
yeah, it may have both static and dynamic members at the same time